cli/types/tests/plugins-test.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Test that plugin authors can write .d.ts files to extend Cypress types
+ * Unfortunately we cannot test that vendor types located in node_modules are working
+ * since those are copied as part of the deploy process
+ *
+ */
```

Comment:
this needs more documentation - this is what someone who writes a new command or adds a property to `cy` global object should do right. So we should describe this better, even for ourselves.

Also, the formatting is all off below, maybe it needs a prettier ...